gen3-client/g3cmd/download-multiple.go

```diff
@@ -17,7 +19,110 @@ import (
 	"github.com/uc-cdis/gen3-client/gen3-client/jwt"
 )
 
-func batchDownload(numParallel int, reqs []*grab.Request) {
+func lastString(ss []string) string {
```

Comment:
since you only use this function once and it's a one-liner, i'm thinking you could just use the one line